pterodactyl.php

```diff
@@ -91,6 +91,36 @@ public function getGroupOrderOptions()
         ];
     }
 
+    /**
+     * Determines which module row should be attempted when a service is provisioned
+     * for the given group based upon the order method set for that group.
+     *
+     * @return int The module row ID to attempt to add the service with
+     * @see Module::getGroupOrderOptions()
+     */
+    public function selectModuleRow($module_group_id)
```

Comment:
I'm not sure how adding this method solves the problem described in issue #19. The abstract Module class implements this method already and supports more than the "first" group order.

It seems like the problem may be more related to the issue of retrieving the current module row set on the module when making an API request (which is where the error "An internal error occurred..." comes from). 

It looks like this module overrides the selected module row by calling `setModuleRow` in several places. Very few modules need to call that method, and I'm not sure this module is one of them.